scripts/mimc_constraints.sage

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/sage
```

Comment:
What about moving this script in `libzeth/circuits/mimc`, next to https://github.com/clearmatics/zeth/blob/mimc-configurations/libzeth/circuits/mimc/mimc_round_constants_generation.py?

Reply:
I'd prefer to keep these scripts in one place, rather than scattered amongst the source. It makes them less easy to find, and it's not clear to me that a mimc script belongs here rather than with the python or solidity implementation.  I realise it's not the same for the round constants script which is currently only used in the circuit implmentation (and in fact I hadn't noticed it there), but I'd rather move this into `/scripts` (or some subdirectory) unless you feel strongly.

Let me know.

Reply:
(Added a commit to move the script to `scripts` so all scripts are there, but can easily be undone if you prefer to keep it where it was.)

Reply:
Sure, grouping them all in "scripts" is fine by me. That being said I think we would benefit from having more structure in the `scripts` folder. I added a `docker` subdirectory some time ago that aims to be the target for the docker entrypoints etc. I would do the same for the `test_` scripts: `cd scripts && mkdir tests && cp test_<NAME>.sh tests/<NAME>.sh`, and the same for `sage` scripts.
That's for another ticket.

Reply:
See #314